wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UserInputRequest+LegalHold.swift

```diff
@@ -28,9 +28,10 @@ extension UserType where Self: SelfLegalHoldSubject {
      * - parameter inputHandler: The block to execute with the password of the user.
      * - note: If the user dismisses the alert, we will make the legal hold request as acknowledged.
      */
-
     func makeLegalHoldInputRequest(for request: LegalHoldRequest, cancellationHandler: @escaping () -> Void, inputHandler: @escaping (String?) -> Void) -> UserInputRequest {
+        // FIXME: [F] move to useCase
```

Comment:
FIXME: move this to usCase in order to make fingerprint async

Reply:
Refactored to inject the fingerprint